grpc-messenger-service/src/main/java/com/kucuk/server/MessageServer.java

```diff
@@ -59,12 +73,33 @@ private void blockUntilShutdown() throws InterruptedException {
         }
     }
 
+    private ManagedChannel getChannel() throws SSLException {
```

Comment:
getChannel is too generic, and this (createChannelForBlockingCallToTheOtherService) might be too long. :)